core-rust-bridge/src/main/java/com/radixdlt/environment/NodeConstants.java

```diff
@@ -115,8 +115,8 @@ public final class NodeConstants {
           .toNonNegativeLong()
           .unwrap();
 
-  public static final int DEFAULT_COST_UNIT_LIMIT =
-      Natives.builder(NodeConstants::getDefaultCostUnitLimit)
+  public static final int DEFAULT_EXECUTION_COST_UNIT_LIMIT =
```

Comment:
@malexandru-rdx I touched this line only to make it compile.
But the Engine also introduced "finalization cost unit limit" next to this one. Do you want me to somehow reflect this change in Node config?
Is it gonna be trivial enough to fit in this PR?